Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN addgroup --gid 2000 --system appgroup && \
 WORKDIR /app
 COPY --from=builder --chown=appuser:appgroup /app/build/libs/calculate-journey-variable*.jar /app/app.jar
 COPY --from=builder --chown=appuser:appgroup /app/build/libs/applicationinsights-agent*.jar /app/agent.jar
-COPY --from=builder --chown=appuser:appgroup /app/AI-Agent.xml /app
+COPY --from=builder --chown=appuser:appgroup /app/applicationinsights.json /app
```

Comment:
This change was needed as part of pulling in the latest dps gradle spring boot plugin.